src/app/parcels/form/ParcelForm.tsx

```diff
@@ -67,7 +67,7 @@ export const initialParcelFields: ParcelFields = {
 export const initialParcelFormErrors: FormErrors = {
     voucherNumber: Errors.none,
     packingDate: Errors.initial,
-    packingTime: Errors.initial,
+    packingSlot: Errors.initial,
```

Comment:
I'm not sure I understand this `initialParcelFormErrors` here. Since Errors.initial is `"required"`, does that mean we show the "required" error on opening the form?

Reply:
The errors are only shown after the submit button is clicked. If there is a change in the input fields, error is set to Errors.none.